packages/uicore/src/components/MultiTypeInput/MultiTypeInput.tsx

```diff
@@ -234,7 +236,7 @@ export function ExpressionAndRuntimeType<T = unknown>(props: ExpressionAndRuntim
           }}
           name={name}
           items={expressions}
-          inputProps={{ placeholder: placeholder || EXPRESSION_INPUT_PLACEHOLDER }}
```

Comment:
I see that the component already takes a `placeholder` prop and sets it on the input, without any conditions. Why do we need to add it again?

Reply:
placeholder prop is used for regular view  and runtime view too. 
This might conflict as we want to have a special place holder for the expression view 
to avoid confusion i have used a separate prop